File: _java/java_basic/src/main/java/ch16/sec01/LambdaExample.java

package ch16.sec01;

public class LambdaExample {
    public static void main(String[] args) {
// x + y 계산
// x – y 계산
    }

    public static void action(Calculable calculable) {
//데이터
        int x = 10;
        int y = 4;
//데이터 처리
        calculable.calculate(x, y);
    }
}

File: _java/java_basic/src/main/kotlin/ch16/sec01/LambdaExample.kt
package ch16.sec01

object LambdaExample {
    @JvmStatic
    fun main(args: Array<String>) {
        // x + y 계산
        // x – y 계산
    }

    fun action(calculable: Calculable) {
        //데이터
        val x = 10
        val y = 4
        //데이터 처리
        calculable.calculate(x, y)
    }
}